README.md

```diff
@@ -1,158 +1,552 @@
+
+### Dojofile options
+
+`Dojofile` has several settings to control `dojo` behavior.
+
+##### Dojo driver
+
+```toml
+DOJO_DRIVER="docker"
+```
+Defines which driver to use. Possible values are [docker](#docker-driver) or [docker-compose](#docker-compose-driver). Default is `docker`.
+
+*equivalent CLI option is: `-driver`*
+
+##### Image
+
+```toml
+DOJO_DOCKER_IMAGE="kudulab/dotnet-dojo:3.1.0"
+```
+Defines which image to use. The value must be a valid docker image reference, same as you would specify in `docker pull`. There is no default, you must specify an image in Dojofile or in CLI arguments.
+
+*equivalent CLI option is: `-image`*
+
+##### Docker options
+
+```toml
+DOJO_DOCKER_OPTIONS="-p 9090:80 --privileged"
+```
+Defines additional arguments for the `docker run` command. Default is empty.
+
+*equivalent CLI option is: `-docker-options`*
+
+##### Outer working directory
+
+```toml
+DOJO_WORK_OUTER="/tmp/my-project"
+```
+Defines directory on the docker host to mount into the container at `/dojo/work` (by default).
+The default is to mount current working directory.
+
+`Dojo` setups following mount `DOJO_WORK_OUTER:DOJO_WORK_INNER`.
+
+*equivalent CLI option is: `-work-dir-outer`*
+
+We do not recommend changing this.
+
+##### Inner working directory
+
+```toml
+DOJO_WORK_INNER="/dojo/work/my-project"
+```
+Defines directory inside the docker container, which is mounted from the host. By default `/dojo/work`.
+
+`Dojo` setups following mount `DOJO_WORK_OUTER:DOJO_WORK_INNER`.
+
+*equivalent CLI option is: `-work-dir-inner`*
+
+We do not recommend changing this.
+
+##### Identity directory
+
+```toml
+DOJO_IDENTITY_OUTER="/home/joe"
 ```
-$ dojo --image=alpine:3.8 whoami
-2019/01/25 13:50:37 [ 1]  INFO: (main.main) Dojo version 0.1.0
-2019/01/25 13:50:37 [ 7]  INFO: (main.DockerDriver.HandleRun) docker command will be:
- docker run --rm -v /home/ewa/code/dojo:/dojo/work -v /home/ewa:/dojo/identity:ro \
-   --env-file=/tmp/dojo-environment-dojo-dojo-2019-01-25_13-50-37-28841830 \
-   -ti --name=dojo-dojo-2019-01-25_13-50-37-28841830 alpine:3.8 whoami
-root
+Defines directory on the docker host to mount into the container at `/dojo/identity`.
+The default is to mount current user's home. See more about [identity directory](#home-and-identity-directory).
+
+`Dojo` setups following mount `DOJO_IDENTITY_OUTER:/dojo/identity`.
+
+*equivalent CLI option is: `-identity-dir-outer`*
+
+##### Blacklist variables
+
+```toml
+DOJO_BLACKLIST_VARIABLES="HOME,USER,MY_PREFIX_*"
 ```
+By default dojo will pass environment variables from host to the container.
+`DOJO_BLACKLIST_VARIABLES` can be used to control which variables should not be mapped.
+It is a list of environment variables, split by commas, which should not be transfered from host to the docker container.
```

Comment:
Could you please also add that a blacklisted variable name can end with an asterisk, e.g. `BASH*`, which means that all the variables with `BASH` prefix (like `BASH_123`, `BASHBLA`) will be blacklisted?